string start_state
string wait_state
---
bool can_start
